Backend/DataAccessLayer/Repositories/ClaimRepository.cs

```diff
@@ -38,27 +38,26 @@ public Service GetService(string claimName)
             }
         }
 
-        public void AddServiceToUser(Guid userId, Guid serviceId, Guid subjectUserId)
+        public void AddServiceToUser(Guid userId, Guid serviceId)
         {
             using (var _db = new DatabaseContext())
             {
                 var u = new Claim
                 {
                     UserId = userId,
-                    ServiceId = serviceId,
-                    SubjectUserId = subjectUserId
+                    ServiceId = serviceId
                 };
                 _db.Claims.Add(u);
                 _db.SaveChanges();

```

Comment:
Weren't we going to pass in the _db variable in order to keep SaveChanges() in the manager layer

Reply:
Same for line 21 in this class